modules/xbox360/LICENSE.txt

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2013-2015 Emilio López and individual contributors.
```

Comment:
Hmm, this is quite interesting to me. This module has never lived as a unique (like `gmake2`, `raw` and `self-test`) and probably should just default to the project license. I don't really know how the legal side of all this works, but it seems strange to me that you can drop files into a project with a different copyright holder. Additionally, a (large?) portion of this code already existed under the project license and I'm pretty sure you can't just change the license on it because you moved it to another file in a different folder.

Also, the copyright years is wrong, it should probably be `2018-2018` or just `2018`.

I'm not trying to be a jerk here, I really don't know how the legality of all this works but it just seems really strange to me. @starkos I would imagine you probably have a better understanding of this? I've always just put it down as `Jason Perkins and …` when I create new files, I assumed it was supposed to be the project copyright notice, not who creates the file?